src/frontend/src/pages/FinancePage/FinanceComponents/GenerateReceiptsModal.tsx

```diff
@@ -43,19 +49,28 @@ const GenerateReceiptsModal = ({
   const [startDatePickerOpen, setStartDatePickerOpen] = useState(false);
   const [endDatePickerOpen, setEndDatePickerOpen] = useState(false);
```

Comment:
ok wait, all these values should just be in the form rather than in useStates, having them in both is overkill. Also if we do everything through the form, we can just set the defaultValues on the useForm (see how we do it in the reimbursement request form)

Reply:
I did not make those changes, but I can create default values within the form for these if you want.

Reply:
yeah I know, but we should be doing this code the right way while you're in here, so that it's better later on when we expand this modal a lot (which I think we will)